src/index.html

```diff
@@ -11,8 +11,57 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100..900;1,100..900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      data-qa="card"
+      class="product"
+    >
+      <img
+        class="product__photo"
+        src="./images/imac.jpeg"
+        alt="imac"
+      />
+      <h2 class="product__name">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+      <div class="product__code">Product code: 195434</div>
+
+      <div class="product__rate">
+        <ul class="stars">
```

Comment:
```suggestion
        <ul class="stars stars--4">
```
Consider using the HTML structure from the previous task